js/ui/components/trayArea.js

```diff
@@ -118,15 +118,15 @@ var TrayArea = new Lang.Class({
 
         // Saturation
         if (arguments.length == 1) {
-            let sat_effect = new Clutter.DesaturateEffect({ factor : ICON_SATURATION });
-            sat_effect.set_factor(ICON_SATURATION);
-            sat_effect.set_factor(ICON_SATURATION);
+            let sat_effect = new Clutter.DesaturateEffect({ factor : ICON_DESATURATION });
+            sat_effect.set_factor(ICON_DESATURATION);
+            sat_effect.set_factor(ICON_DESATURATION);
```

Comment:
Why is this line duplicated?

Reply:
Looks like an error we inherited from the original extension:
https://github.com/phocean/TopIcons-plus/blob/master/extension.js#L328